pyproject.toml

```diff
@@ -13,7 +13,7 @@ dependencies = [
   "inspect-ai>=0.3.104",
   # pin litellm so that we know what model costs we're using
   # see https://github.com/allenai/astabench-issues/issues/391 before changing
-  "litellm==1.75.8",
+  "litellm<=1.75.8",
```

Comment:
Don't go above this, to avoid silently supporting new models (which could lead to [this scenario](https://github.com/allenai/agent-eval/pull/63#discussion_r2289051377)).

Reply:
does the dep conflict in astabench come from the sqa deps? In which case, the fundamental issue is that we're picking up their pinned version and will likely encounter errors in the future whenever they update the version in their lib

not sure what to do about it... but I do think for now we could pin to `==1.68.0`  to satisfy (but still be sure of what version we're getting; we don't want 0.2.0 for example)